### PR TITLE
Allow key files to be symlinks

### PR DESCRIPTION
**Pull request by strugee**
Before the qrexec service would mistakenly think there were no available keys.

Note: I'm not really sure what the second `find` call does... so I'm not sure if it needs `-L` too. What's the intention there?

### CODE REVIEW

**Comment by mtdcr** [comment]
Looks good. The second `find` finds nonempty subdirectories, which are treated as keyrings. We could use `-L` there, too, but I don't think it's important. We could also wait until someone needs it.

**Comment by strugee** [comment]
Guess we might as well. I'll push an amended commit real fast.

Thanks for the explanation, `find` commandlines are so dense and hard to parse sometimes :P

**Comment by strugee** [comment]
Done.

**Comment by mtdcr** [comment]
Thank you!

**Comment by strugee** [comment]
Absolutely.
